pytorch_lightning/core/lightning.py

```diff
@@ -159,6 +159,14 @@ def on_gpu(self):
         """
         return self.device.type == "cuda"
 
+    @property
+    def automatic_optimization(self) -> bool:
+        """
+        If False you are responsible for calling .backward, .step, zero_grad.
+        Meant to be used with multiple optimizers by advanced users.
```

Comment:
I think the above line is sufficient. A noob user can use it with a single optimizer too.
```suggestion
```